engine/common/requester/engine.go

```diff
@@ -59,9 +59,9 @@ func New(log zerolog.Logger, metrics module.EngineMetrics, net network.Network,
 	cfg := Config{
 		BatchThreshold:  32,
 		BatchInterval:   time.Second,
-		RetryInitial:    4 * time.Second,
-		RetryFunction:   RetryGeometric(2),
-		RetryMaximum:    2 * time.Minute,
+		RetryInitial:    1 * time.Second,
+		RetryFunction:   RetryLinear(1 * time.Second),
+		RetryMaximum:    10 * time.Second,
```

Comment:
This is changing the default config for the `requester` engine in general, which is used for several different entity types. We only want to modify the config for the engine configured with the `request-collections` channel, which is set up here: https://github.com/onflow/flow-go/blob/master/cmd/execution_builder.go#L742-L751.

We should also be sure to change the `BatchInterval` (set to 1m in the link above). `BatchInterval` is the interval at which the engine will send off requests, without outside intervention. `Retry*` configs are used to decide whether to include a particular item in a batch request, not when to submit the request. So with a `BatchInterval` of 1m and a 1s retry, we may still be sending requests once per minute.

Reply:
> So with a `BatchInterval` of 1m and a 1s retry, we may still be sending requests once per minute.

Good point!  `matchOrRequestCollections` does force the re-try almost constantly, so we should not retry on the batch interval too much. That said, lowered since 1 minute seems like an overkill.